deisctl/units/deis-swarm-manager.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=deis-cache
+
+[Service]
+EnvironmentFile=/etc/environment
+TimeoutStartSec=20m
+ExecStartPre=/bin/sh -c "docker history smothiki/swarm >/dev/null || docker pull smothiki/swarm"
```

Comment:
Is there something necessary in your custom image that docker's `swarm` image doesn't provide?